src/modules/Dropdown/Dropdown.js

```diff
@@ -184,21 +188,19 @@ export default class Dropdown extends Component {
 
   componentWillReceiveProps(nextProps) {
     super.componentWillReceiveProps(nextProps)
-    // console.debug('Dropdown.componentWillReceiveProps()')
-    // console.log('changed props:', _.transform(nextProps, (res, val, key) => {
-    //   if (val !== this.props[key]) res[key] = val
-    // }, {}))
+    debug.groupCollapsed('componentDidUpdate()')
+    debug('changed props:', objectDiff(nextProps, this.props))
```

Comment:
This objectDiff will still happen unless the debug statements are actually stripped from the source code. `makeDebugger` simply won't log them unless told, but the arguments are still evaluated. You could consider passing it a callback function so that the inner content is lazily evaluated (i.e. then only runs in !prod).